cmd/mssqldef/mssqldef.go

```diff
@@ -53,8 +53,17 @@ func parseOptions(args []string) (database.Config, *sqldef.Options) {
 	}
 
 	desiredFile, currentFile := sqldef.ParseFiles(opts.File)
+
+	var desiredDDLs string
+	if !opts.Export {
+		desiredDDLs, err = sqldef.ReadFile(desiredFile)
```

Comment:
Given that we resort to `log.Fatalf` anyway, you could call `log.Fatalf` in `sqldef.ReadFile` and only return a string. Then you should also be able to do `options.DesiredDDLs = sqldef.ReadFile(desiredFile)` after defining `options` so that you can skip defining an extra local variable `desiredDDLs`.

Reply:
`sqldef.ReadFile` is also used in other parts of the code so I would like to fix it in another PR.